### DISCUSSION TITLE
Treatment dialogue is cut and not usable on Tickwatch 5 Pro

### DISCUSSION THREAD

**Discussion by xannasavin** [NightscoutFoundation/xDrip#3191] [Bug reports]
I got a new Ticwatch 5 Pro and installed xDrip - worked like a charm.
I came to notice, that the dialogue for the treatment is not fitting the screen, in fact it is cut in a way that makes entering treatments pretty hard.
Since it was correct on my former Ticwatch 3 Pro that also had a round screen, I assume it must be either related to a different resolution that is not correctly supported or the new WearOS 3?
![PXL_20231117_062224413](https://github.com/NightscoutFoundation/xDrip/assets/48352696/e53b0fe7-3162-4608-bb7e-62d456d99cbf)
![PXL_20231117_062239606](https://github.com/NightscoutFoundation/xDrip/assets/48352696/2b424fbf-f48f-44cf-a608-1a1fd30c2868)
Possible to resize the dialogue just a bit, so it will be possible again to easily touch all the buttons?

(Removed the link to the treatment dialogue improvements since it was mainly for phone, not for the watch)

**Comment by Navid200** [comment]
Unfortunately, no one is currently addressing Android Wear concerns.  
There are individuals who have the experience.  I wish they would step in and make suggestions or open PRs.
Without that, since we are all volunteers, there is not much we can do.

**Reply by darkteckno** [reply]
Seriously is this all the feedback? Ticwatch used to be the goto for xdrip. It's just the same story as when I bought a fossil watch.

**Reply by Navid200** [reply]
Used to be, that's correct.  A lot has changed.  xDrip is supported by volunteers.  Volunteers come and go.  We have no way of demanding from a volunteer who stops to contribute to come back and help us with something.
I have no experience with WearOS.  What do you expect me to do?

**Reply by darkteckno** [reply]
I wasn't meaning you personally just a general observation.
